package.json

```diff
@@ -14,8 +14,10 @@
     "test": "jest",
     "test:ci": "yarn lint && jest --runInBand && yarn build",
     "test:watch": "jest --watch",
-    "examples:ecommerce": "./scripts/run-ecommerce.sh",
-    "examples:ecommerce:build": "./scripts/build-ecommerce.sh",
+    "examples:ecommerce": "./scripts/run-example.sh e-commerce",
+    "examples:ecommerce:build": "./scripts/build-example.sh e-commerce",
```

Comment:
The build script wasn't renamed to `build-example` @iam4x

Reply:
Yes I did not updated it yet :)